shared/src/main/java/net/blay09/mods/balm/api/event/ChunkEvent.java

```diff
@@ -0,0 +1,42 @@
+package net.blay09.mods.balm.api.event;
+
+import net.minecraft.world.level.ChunkPos;
+import net.minecraft.world.level.LevelAccessor;
+import net.minecraft.world.level.chunk.ChunkAccess;
+
+public abstract class ChunkEvent {
```

Comment:
I think it's better to stay away from super generic names and have more "dedicated" events instead, since there's so many things that could fall under a "chunk event" and blow the class up.

Maybe could call it `ChunkLoadingEvent.Load/Unload` or just two seperate `ChunkLoadedEvent`/`ChunkUnloadedEvent` instead.